nbstripout.py

```diff
@@ -211,6 +243,11 @@ def install(attrfile=None):
     check_call(['git', 'config', 'filter.nbstripout.clean', filepath])
     check_call(['git', 'config', 'filter.nbstripout.smudge', 'cat'])
     check_call(['git', 'config', 'filter.nbstripout.required', 'true'])
+    try:
+        # don't override global config
+        check_output(['git', 'config', 'filter.nbstripout.extrakeys'])
```

Comment:
I don't quite understand what this is needed for?

Reply:
removed